qiskit_nature/second_q/operators/__init__.py

```diff
@@ -53,6 +53,7 @@
     "SecondQuantizedOp",
     "SpinOp",
     "VibrationalOp",
+    "build_dual_index",
```

Comment:
I will redo this slightly and make `build_dual_index` a `staticmethod` of `VibrationalOp` instead. That shows a lot better that this method is a specific utility for that class.